src/config.ts

```diff
@@ -7,11 +7,13 @@ const CONFIG_FILE_NAME = '.soycriticrc';
 export interface Config {
   callToImportRegex: string
   callToImportReplace: string
+  implicitParams: any
```

Comment:
Try something like this:
```ts
export interface Config {
  implicitParams: {
    [nameOrRegex: string]: string | Array<string>
  }
}
```
It definitely doesn't make sense to parse JSON at run time and have a type-safe interface created. I know how other strongly typed languages accomplish this (Elm and JSON decoders), but I'm not sure about typescript. I'll look into it.